### ISSUE TITLE
Reported OPs

### ISSUE THREAD

**Issue by HatemSelim94** [liuzechun/ReActNet#23]
Thank you for your great work. I have a question about the reported OPs. When calculating BOPs, convolution layers are the only layers considered and In FLOPS calculation, other layers such as (Batch norm, activation(Binarization of inputs), skip connections(Addition), ...) are considered, is that correct?
Thank you for your time.

**Comment by liuzechun** [comment]
Hi, thanks for your interest in our work! Yes, that’s true. The FLOPs in the full precision modules (e.g. Batchnorm2d, PReLU, AvgPool) are not considered.
